adapters/eplanning/eplanning_test.go

```diff
@@ -0,0 +1,193 @@
+package eplanning
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"bytes"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"testing"
+	"time"
+)
+
+type aBidInfo struct {
+	deviceIP string
+	deviceUA string
+	tags     []aTagInfo
+	width    uint64
+	height   uint64
+	buyerUID string
+	delay    time.Duration
+}
+
+type aTagInfo struct {
+	code     string
+	bidfloor float64
+	instl    int8
+	price    float64
+	content  string
+	dealId   string
+}
+
+func TestOpenRTBRequest(t *testing.T) {
+	bidder := new(EPlanningAdapter)
+	bidder.URI = "http://e-planning-net"
+	testData := createTestData()
+	request := createOpenRtbRequest(testData)
+
+	httpRequests, errs := bidder.MakeRequests(request)
+
+	if len(errs) > 0 {
+		t.Errorf("Got unexpected errors while building HTTP requests: %v", errs)
+	}
+	if len(httpRequests) != 1 {
+		t.Fatalf("Unexpected number of HTTP requests. Got %d. Expected %d", len(httpRequests), 1)
+	}
+
```

Comment:
You don't seem to do any checks that you generated a valid ePlanning request. If you had, you would likely have caught my issues above. Moving to the `.json` file tests will provide this, as they will test against the expected request to be sent to your adapter as defined in your `.json` test file.